lib/releaselib.sh

```diff
@@ -57,11 +57,10 @@ release::update_job_cache () {
     # the cache should be complete after that
     [[ -n ${JOB[$run]} ]] && break
 
-    buildlog=$($GSUTIL cat $logroot/$job/$run/build-log.txt 2>/dev/null)
-    if [[ "$buildlog" =~ Uploading\ build\ result:\ SUCCESS ]] &&
-       [[ "$buildlog" =~ \
-          (build_version=|Found Kubernetes version: )(${VER_REGEX[release]}\.${VER_REGEX[build]}) ]]; then
-      JOB[$run]=${BASH_REMATCH[2]}
+    buildstate=$($GSUTIL cat $logroot/$job/$run/*.json 2>/dev/null)
+    if [[ "$buildstate" =~ \"result\":\ \"SUCCESS\", ]] &&
```

Comment:
I had used jq previously but that also included more gsutil calls and more piping and executing of external tools (jq).
This current implementation makes a single gsutil call and then the rest is regex'ing in bash.  This is the most optimized way of doing it.